components/miller/stories/miller.stories.js

```diff
@@ -17,27 +17,25 @@ export default {
 		rootClass: "spectrum-MillerColumns",
 		columns: [
 			{
+				isSelectable: false,
```

Comment:
since these were all set to true before, do we want to keep them as true?

Reply:
We can but I noticed we weren't really getting a full view of all the state combinations for the component so I tried to expand this a bit.